src/backend/api/reviews.js

```diff
@@ -0,0 +1,89 @@
+const { response } = require('express');
+const express = require('express');
+const { request } = require('../app');
+const router = express.Router();
+const knex = require('../database');
+
+router.get('/', async (request, response) => {
+    try {
+        const allReviews = await knex('review').select('*')
+         if(allReviews.length === 0){
+          response.status(404).send("No reviews available !")
+         }else{
+          response.send(allReviews);
+         }
+        } catch (error) {
+          throw response.status(404).send(error);
+        }
+})
+
+//	POST
+
+router.post("/", async (request, response) => {
+    try {
+      const [insertedReviews] = await knex('review').insert({
+        id: request.body.id,
+        title: request.body.title,
+        description: request.body.description,
+        meal_id: request.body.meal_id,
+        stars: request.body.stars,
+        created_date: request.body.created_date,
+      });
+      const reviews = await knex("review").select("*");
+      if(insertedReviews != 0){
+        response.send({message: "New review", insertedId : insertedReviews, Reviews: reviews})
+      }
+    } catch (error) {
+      throw response.status(404).send(error);
+    }
+  });
+
+//GET /api/reviews/:id
+router.get("/:id", async (request, response) => {
+    try {
+     // knex syntax for selecting things. Look up the documentation for knex for further info
+     const id = request.params.id ;
+     const reviews = await knex("review").select("*").where('id', id);
+     if(reviews.length === 0){
+      response.status(404).send("No reviews available !")
```

Comment:
Here you're responding with text and not JSON. I would recommend something like this:

```suggestion
      response.status(404).send({ error: "Review not found" })
```

Reply:
Yes will update it